compiler/qsc_frontend/src/typeck/tests.rs

```diff
@@ -4695,3 +4695,52 @@ fn field_access_not_ident() {
         "##]],
     );
 }
+
+#[test]
+fn stressing_generic_inference() {
+    check(
+        indoc! {r#"
+            namespace Test {
+                function swap<'T1, 'T2>(op : ('T1, 'T2) -> ('T2, 'T1), input: ('T1, 'T2)) : ('T2, 'T1) {
+                    op(input)
+                }
+                function nested_swaps() : Unit {
+                    // let a = swap(swap(swap(swap(swap(swap(swap(swap(swap(swap(swap(swap(swap(swap(swap(_)))))))))))))));
+                    let a = swap(swap(swap(swap(_))));
```

Comment:
Looks like you have two versions of the test here, with the significantly longer one commented out.

Reply:
Nope, left behind by mistake.